eox_tagging/api/v1/__init__.py

```diff
@@ -0,0 +1,6 @@
+"""
+API initialization module
+"""
+from eox_tagging.api.v1.permissions import load_permissions
+
+load_permissions()
```

Comment:
why do we need to do this ?

Reply:
I think is to create for the first time the permissions for eox_tagging. This is what's done in eox-core